examples/index.html

```diff
@@ -44,17 +44,26 @@
   <link href="../node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
   <link href="../node_modules/bootstrap-colorpicker/dist/css/bootstrap-colorpicker.css" rel="stylesheet">
 
-  <link rel="stylesheet" href="demo.css">
+  <link id="stylesheet" rel="stylesheet" href="demo.css">
   <link href="../node_modules/font-awesome/css/font-awesome.min.css" rel="stylesheet">
   <link href="../node_modules/selectize/dist/css/selectize.default.css" rel="stylesheet">
   <!-- for crop module: -->
   <!-- for crop module: -->
   <link href="../node_modules/imgareaselect/distfiles/css/imgareaselect-default.css" rel="stylesheet">
   <link href="./selectize.default.css" rel="stylesheet">
-  <link rel="stylesheet" href="demo.css">
 
   <div class="container-fluid">
 
+    <nav>
+    <div class="theme-switch-wrapper">
+      <label class="theme-switch" for="checkbox">
+        <input type="checkbox" id="checkbox" />
+        <div class="slider round"></div>
+      </label>
+      <p class="toggleTitle"><i>Enable Dark Mode!</i></p>
```

Comment:
Please use hyphens instead of camel case in the class name (just to maintain consistency).